src/OrleansAWSUtils/Options/DynamoDBGatewayProviderOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrleansAWSUtils.Options
+{
+    public class DynamoDBGatewayProviderOptions
+    {
+        /// <summary>
+        /// Connection string for DynamoDB Storage
+        /// </summary>
+        public string ConnectionString { get; set; }
```

Comment:
As we are using strongly typed options now, we should keep the connectionstring parsing only for legacy and accept DynamoDB native options here like: ```AmazonDynamoDBConfig```, separate ```AccessKey``` and ```Secret``` properties. having a Func<AmazonDynamoDBClient> as part of options is also a good solution, similarly to how other "Provider" configurations are done in AspNetCore.